primer3/argdefaults.py

```diff
@@ -42,5 +53,168 @@ class Primer3PyArguments:
     temp_only: int = 0
     calc_type_wrapper = 'ANY'
 
+    dmso_conc: float = 0.0
+    dmso_fact: float = 0.6
+    formamide_conc: float = 0.8
+    annealing_temp_c: float = 50.0  # 70.0 or 120 in primer3 source
+
     def todict(self) -> dict:
         return dataclasses.asdict(self)
+
+
+# ~~~~~~~ RUDIMENTARY PRIMER3 MAIN WRAPPER (see Primer3 docs for args) ~~~~~~ #
+def format_boulder_io(p3_args: Dict[str, Any]) -> bytes:
+    '''Convert argument dictionary to boulder formatted bytes
+
+    Args:
+        p3_args: primer3 arguments to format boulder style
+
+    Returns:
+        Boulder formatted byte string
+    '''
+    class TransformT:
+        comma_pair_list_space_separated = 'comma_pair_list_space_separated'
+        dashed_range = 'dashed_range'
+        path_fixer = 'path_fixer'
+        semicolon = 'semicolon'
+        space_separated = 'space_separated'
+        tuple_list_comma_separated = 'tuple_list_comma_separated'
```

Comment:
I'm having trouble understanding the difference (even after looking at the source) between `comma_pair_list_space_separated` and `tuple_list_comma_separated`. I'm sure I'm just missing something -- looks like the expected `BoulderIO` output should be the same?

Reply:
There's a difference. 
`comma_pair_list_space_separated` is `1,2 3,4 5,6`
`tuple_list_comma_separated` is `1,2,3,4,5,6`

I went and ahead and removed this in the refactor to `wrap` and `unwrap` versions